test/unit/eval_algorithms/test_factual_knowledge.py

```diff
@@ -90,6 +114,8 @@ class TestCaseFactualKnowledgeEvaluateSample(NamedTuple):
                 model_input="Who is Andrew R. Jassy?",
                 model_output="Chief Executive Officer of Amazon.com Inc.",
                 target_output="Chief Executive Officer of Amazon.com, Inc.",
+                delimiter="<OR",
```

Comment:
Delimiter is different here (doesn't matter functionally but could be confusing?)

Reply:
@polaschwoebel We will work towards to change target_output (ground_truth) a list as the Ray issue is solved, then the delimiter should go away and there won't be conflict.
https://github.com/aws/fmeval/pull/305

Reply:
Makes sense. Here, I simply meant "<OR>" vs "<OR" (small typo in the code above).